physx/src/visual_debugger.rs

```diff
@@ -53,24 +53,36 @@ impl VisualDebugger {
     /// This function internally calls `new_with_timeout` with a default timeout
     /// of 10 ms.
     pub fn new(foundation: &mut impl Foundation, port: i32) -> Option<Self> {
-        VisualDebugger::new_with_timeout(foundation, port, 10)
+        VisualDebugger::new_with_timeout(foundation, "localhost", port, 10)
+    }
+
+    /// Create a new `VisualDebugger` instance, a utility class that
+    /// combines the TCP setup and the Pvd into one object. The port
+    /// default for the PVD program is port 5425, so it is suggested to use this
+    /// unless you're explicitly changing the other one as well.
+    ///
+    /// This function internally calls `new_with_timeout` with a default timeout
+    /// of 10 ms.
+    pub fn new_remote(foundation: &mut impl Foundation, host: &str, port: i32) -> Option<Self> {
```

Comment:
Could maybe make host: `Option<&str>`, where None means localhost. But this works too!

Reply:
I think that'd make all code more complex.